specification/cognitiveservices/data-plane/ComputerVision/v1.0/ComputerVision.json

```diff
@@ -0,0 +1,1482 @@
+                "operationId": "GenerateThumbnail",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/octet-stream"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/SubscriptionKey"
+                    },
+                    {
+                        "name": "width",
+                        "type": "integer",
+                        "in": "query",
+                        "required": true,
+                        "minimum": 1,
+                        "maximum": 1023,
+                        "description": "Width of the thumbnail. It must be between 1 and 1024. Recommended minimum of 50."
+                    },
+                    {
+                        "name": "height",
+                        "type": "integer",
+                        "in": "query",
+                        "required": true,
+                        "minimum": 1,
+                        "maximum": 1023,
+                        "description": "Height of the thumbnail. It must be between 1 and 1024. Recommended minimum of 50."
+                    },
+                    {
+                        "$ref": "../../Common/Parameters.json#/parameters/ImageUrl"
+                    },
+                    {
+                        "name": "smartCropping",
+                        "type": "boolean",
+                        "in": "query",
+                        "required": false,
+                        "default": false,
+                        "description": "Boolean flag for enabling smart cropping."
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "schema": {
+                            "type": "file"
+                        }
+                    },
+                    "default": {
+                        "schema": {
+                            "$ref": "#/definitions/ComputerVisionError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Generate Thumbnail request": {
+                        "$ref": "./examples//SuccessfulGenerateThumbnail.json"
+                    }
+                }
+            }
+        },
+        "/ocr?overload=url": {
+            "post": {
+                "description": "Optical Character Recognition (OCR) detects printed text in an image and extracts the recognized characters into a machine-usable character stream.   Upon success, the OCR results will be returned. Upon failure, the error code together with an error message will be returned. The error code can be one of InvalidImageUrl, InvalidImageFormat, InvalidImageSize, NotSupportedImage,  NotSupportedLanguage, or InternalServerError.",
+                "operationId": "RecognizePrintedText",
```

Comment:
`operationId` must be `groupname_methodname` -- 

`groupname` is used to create a class in most generated languages
`methodname` is used to generate a method name in most languages